packages/run-protocol/src/proposals/econ-behaviors.js

```diff
@@ -457,6 +460,11 @@ export const startVaultFactory = async (
     shortfallInvitationAmount,
   });
 
+  const chainStorage = await chainStorageP;
+  const storageNode = await (chainStorage &&
+    E(chainStorage).getChildNode(STORAGE_PATH));
+  const marshaller = E(board).getPublishingMarshaller();
```

Comment:
I'm on the fence whether you should use `getPublishingMarshaller` here, or just `getReadOnlyMarshaller`. Maybe write a comment as to why it's safe to publish all the ocaps on the board?

Reply:
AMM had no private state and has dynamic brands, so it's plausible for it to end up causing more brands to get registered. Vaults should use a read only publisher so there's no risk of exposing a new cap.